### ISSUE TITLE
iOS 15 Next/Prev Issue

### ISSUE THREAD

**Issue by amachairidis** [hackiftekhar/IQKeyboardManager#1830]
Hello, I'm using a UItableview and each cell has a UItextfield.
Before upgrading to iOS 15 Beta everything worked as expected (next/previous).

After upgrading to iOS 15 Beta this has stopped working. It works sometimes and jumps to random cells, but most of the time it doesn't do anything.

It does show the button correctly though, meaning the first cell has previous disabled, the last cell has next disabled and in-between both are enabled.

iOS 14.7.1 Works
iOS 15 Beta Not Working

My setup is:
Safe Area View > ScrollView > Content View > UITableView (full content size always).
I have tried giving the ContentView IQPreviousNextView class, doesn't change anything.

Anyone has problems with iOS 15 as well?

**Comment by NasrullahKhan** [comment]
@amachairidis any solution? because iOS 15 release and getting this issue on a very big project and client is expecting a build. Please help @hackiftekhar

**Comment by amachairidis** [comment]
> @amachairidis any solution? because iOS 15 release and getting this issue on a very big project and client is expecting a build. Please help @hackiftekhar

No I haven't. Needed to temporarily disable the next/previous for my apps.

**Comment by amachairidis** [comment]
I concluded it was not the libraries fault, although it did work fine/better on pre iOS 15. I had the same problems even without the library. I found that my problem was caused by combination of reloadData and becomeFirstResponder on my table on didEndEditing on the textfields.

I fixed it by changing the reloadData logic to just reload the specific row with the new data using reloadRows([indexpath]) with the current indexpath.
